wrappers/python/nntile/nntile_core.cc

```diff
@@ -343,6 +405,66 @@ void tensor_to_array(const tensor::Tensor<T> &tensor,
     tmp.unregister();
 }
 
+void tensor_to_array(const tensor::Tensor<fp32_fast_tf32_t> &tensor, py::array_t<fp32_t> &array)
```

Comment:
the same with mocking data type can help to reduce code while maintaining readability